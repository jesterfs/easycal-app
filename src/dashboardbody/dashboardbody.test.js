import React from 'react'
import ReactDOM from 'react-dom'
import { BrowserRouter } from 'react-router-dom'
import DashboardBody from './dashboardbody'
import renderer from 'react-test-renderer';

describe('DashboardBody component', () => {
  it('renders without crashing', () => {
    const div = document.createElement('div')
    ReactDOM.render(
      <BrowserRouter>
        <DashboardBody />
      </BrowserRouter>,
      div
    )
    ReactDOM.unmountComponentAtNode(div)
  })
  
  it('renders the UI as expected', () => {
    const div = document.createElement('div')
    const tree = renderer
      .create(<BrowserRouter>
        <DashboardBody />
      </BrowserRouter>,
      div)
      .toJSON();
    expect(tree).toMatchSnapshot();  
    });

});