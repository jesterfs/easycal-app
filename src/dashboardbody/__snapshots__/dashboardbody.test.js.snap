// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`DashboardBody component renders the UI as expected 1`] = `
<div>
  <h3
    className="monthName"
  >
    November
  </h3>
  <div
    className="date-grid"
  >
    <button
      className="calendarSquare"
      style={
        Object {
          "gridColumn": 0,
        }
      }
    >
      <time
        className="dateNumber"
        dateTime="2020-11-01"
      >
        01
      </time>
      <ul />
    </button>
    <button
      className="calendarSquare"
    >
      <time
        className="dateNumber"
        dateTime="2020-11-02"
      >
        02
      </time>
      <ul />
    </button>
    <button
      className="calendarSquare"
    >
      <time
        className="dateNumber"
        dateTime="2020-11-03"
      >
        03
      </time>
      <ul />
    </button>
    <button
      className="calendarSquare"
    >
      <time
        className="dateNumber"
        dateTime="2020-11-04"
      >
        04
      </time>
      <ul />
    </button>
    <button
      className="calendarSquare"
    >
      <time
        className="dateNumber"
        dateTime="2020-11-05"
      >
        05
      </time>
      <ul />
    </button>
    <button
      className="calendarSquare"
    >
      <time
        className="dateNumber"
        dateTime="2020-11-06"
      >
        06
      </time>
      <ul />
    </button>
    <button
      className="calendarSquare"
    >
      <time
        className="dateNumber"
        dateTime="2020-11-07"
      >
        07
      </time>
      <ul />
    </button>
    <button
      className="calendarSquare"
    >
      <time
        className="dateNumber"
        dateTime="2020-11-08"
      >
        08
      </time>
      <ul />
    </button>
    <button
      className="calendarSquare"
    >
      <time
        className="dateNumber"
        dateTime="2020-11-09"
      >
        09
      </time>
      <ul />
    </button>
    <button
      className="calendarSquare"
    >
      <time
        className="dateNumber"
        dateTime="2020-11-10"
      >
        10
      </time>
      <ul />
    </button>
    <button
      className="calendarSquare"
    >
      <time
        className="dateNumber"
        dateTime="2020-11-11"
      >
        11
      </time>
      <ul />
    </button>
    <button
      className="calendarSquare"
    >
      <time
        className="dateNumber"
        dateTime="2020-11-12"
      >
        12
      </time>
      <ul />
    </button>
    <button
      className="calendarSquare"
    >
      <time
        className="dateNumber"
        dateTime="2020-11-13"
      >
        13
      </time>
      <ul />
    </button>
    <button
      className="calendarSquare"
    >
      <time
        className="dateNumber"
        dateTime="2020-11-14"
      >
        14
      </time>
      <ul />
    </button>
    <button
      className="calendarSquare"
    >
      <time
        className="dateNumber"
        dateTime="2020-11-15"
      >
        15
      </time>
      <ul />
    </button>
    <button
      className="calendarSquare"
    >
      <time
        className="dateNumber"
        dateTime="2020-11-16"
      >
        16
      </time>
      <ul />
    </button>
    <button
      className="calendarSquare"
    >
      <time
        className="dateNumber"
        dateTime="2020-11-17"
      >
        17
      </time>
      <ul />
    </button>
    <button
      className="calendarSquare"
    >
      <time
        className="dateNumber"
        dateTime="2020-11-18"
      >
        18
      </time>
      <ul />
    </button>
    <button
      className="calendarSquare"
    >
      <time
        className="dateNumber"
        dateTime="2020-11-19"
      >
        19
      </time>
      <ul />
    </button>
    <button
      className="calendarSquare"
    >
      <time
        className="dateNumber"
        dateTime="2020-11-20"
      >
        20
      </time>
      <ul />
    </button>
    <button
      className="calendarSquare"
    >
      <time
        className="dateNumber"
        dateTime="2020-11-21"
      >
        21
      </time>
      <ul />
    </button>
    <button
      className="calendarSquare"
    >
      <time
        className="dateNumber"
        dateTime="2020-11-22"
      >
        22
      </time>
      <ul />
    </button>
    <button
      className="calendarSquare"
    >
      <time
        className="dateNumber"
        dateTime="2020-11-23"
      >
        23
      </time>
      <ul />
    </button>
    <button
      className="calendarSquare"
    >
      <time
        className="dateNumber"
        dateTime="2020-11-24"
      >
        24
      </time>
      <ul />
    </button>
    <button
      className="calendarSquare"
    >
      <time
        className="dateNumber"
        dateTime="2020-11-25"
      >
        25
      </time>
      <ul />
    </button>
    <button
      className="calendarSquare"
    >
      <time
        className="dateNumber"
        dateTime="2020-11-26"
      >
        26
      </time>
      <ul />
    </button>
    <button
      className="calendarSquare"
    >
      <time
        className="dateNumber"
        dateTime="2020-11-27"
      >
        27
      </time>
      <ul />
    </button>
    <button
      className="calendarSquare"
    >
      <time
        className="dateNumber"
        dateTime="2020-11-28"
      >
        28
      </time>
      <ul />
    </button>
    <button
      className="calendarSquare"
    >
      <time
        className="dateNumber"
        dateTime="2020-11-29"
      >
        29
      </time>
      <ul />
    </button>
    <button
      className="calendarSquare"
    >
      <time
        className="dateNumber"
        dateTime="2020-11-30"
      >
        30
      </time>
      <ul />
    </button>
  </div>
</div>
`;
